guides/data.json

```
@@ -1,4 +1,13 @@
 [
+  {
+    "source": "image-apis"
+  },
+  {
+    "source": "AI-apis"
```

The source should always be in lower case.

```suggestion
    "source": "ai-apis"
```